python/triqs_dft_tools/converters/plovasp/proj_group.py

```diff
@@ -416,8 +416,9 @@ def orthogonalize_projector_matrix(self, p_matrix):
         overlap = np.dot(p_matrix, p_matrix.conj().T)
 # Calculate [O^{-1/2}]_{m m'}
         eig, eigv = np.linalg.eigh(overlap)
-        assert np.all(eig > 0.0), ("Negative eigenvalues of the overlap matrix:"
-           "projectors are ill-defined")
+        eig = np.around(eig,10)  
```

Comment:
Hi @dariofiosca ,
I identified this line as the cause for the failing test. Why introducing here a round to 10 digits? Was this for debugging purpose only? Should we put in the assert again? If I revert these three lines 419-421 back everything seems to work.